dataactbroker/README.md

```diff
@@ -857,6 +857,51 @@ File download or redirect to signed URL
 
 ## File Generation Routes
 
+#### GET "/v1/list_submissions/"
+List submissions for all agencies for which the current user is a member of. Optional query parameters are `?page=[page #]&limit=[limit #]&certified=[true|false]` which correspond to the current page number and how many submissions to return per page (limit). If the query parameters are not present, the default is `page=1`, `limit=5` and if `certified` is not provided, all submissions will be returned containing a mix of the two.
```

Comment:
This doesn't seem to describe the same logic I'm seeing below. In `getSubmissionsByUserId`, aren't we only getting submissions that the current user has submitted (in contrast to "all agencies for which the current user is a member of")?